M2/Macaulay2/packages/PositivityToricBundles.m2

```diff
@@ -480,7 +486,8 @@ if separatesJets(tvb, Verbosity=>opts#Verbosity) >= 0 then true else false;
 --           check if the vector bundle is very ample, using [RJS, Cor. 6.7]
 --   INPUT : 'tvb', toric vector bundle
 --  OUTPUT :  'true' if very ample, otherwise 'false'
-isVeryAmple ToricVectorBundleKlyachko := true >> opts -> tvb -> if separatesJets(tvb, opts) >= 1 then true else false
+--isVeryAmple = method( Options => true ) -- conflicts with Polyhedra
+isVeryAmple (ToricVectorBundleKlyachko) := (tvb) -> if separatesJets(tvb) >= 1 then true else false;
```

Comment:
I think the failing builds are due to this `isVeryAmple` change.  It's a `MethodFunctionWithOptions`, so it should accept options.

Reply:
Thanks for the hint, I hope it's fixed now.